# Snapshot report for `language/directive-prologue/index.js`

The actual snapshot is saved in `index.js.snap`.

Generated by [AVA](https://avajs.dev).

## language/directive-prologue/10.1.1-1-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/10.1.1-10-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/10.1.1-11-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'BinaryExpression operator \'instanceof\' is not supported at this time',
    }

## language/directive-prologue/10.1.1-12-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'NotImplementedException: Cannot use global `eval()` to execute arbitrary code at runtime.',
    }

## language/directive-prologue/10.1.1-13-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'NotImplementedException: Cannot use global `eval()` to execute arbitrary code at runtime.',
    }

## language/directive-prologue/10.1.1-14-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "SyntaxError" is not defined',
    }

## language/directive-prologue/10.1.1-28-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }

## language/directive-prologue/10.1.1-29-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'NotImplementedException: Cannot use `Function` as a constructor to create new functions at runtime.',
    }

## language/directive-prologue/10.1.1-2gs.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/10.1.1-3-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/10.1.1-30-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "SyntaxError" is not defined',
    }

## language/directive-prologue/10.1.1-31-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'NotImplementedException: Cannot use `Function` as a constructor to create new functions at runtime.',
    }

## language/directive-prologue/10.1.1-32-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'NotImplementedException: Cannot use `Function` as a constructor to create new functions at runtime.',
    }

## language/directive-prologue/10.1.1-4-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/10.1.1-5-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "SyntaxError" is not defined',
    }

## language/directive-prologue/10.1.1-5gs.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/10.1.1-6-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/10.1.1-7-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/10.1.1-8-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "SyntaxError" is not defined',
    }

## language/directive-prologue/10.1.1-8gs.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/10.1.1-9-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/14.1-1-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-10-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-11-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-12-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-13-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-14-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-15-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-16-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-17-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-2-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-3-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-4-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-4gs.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/14.1-5-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-5gs.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/14.1-6-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-7-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-8-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/14.1-9-s.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught TypeError: callee is not an object',
    }

## language/directive-prologue/func-decl-final-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/func-decl-inside-func-decl-parse.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/func-decl-inside-func-decl-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "ReferenceError" is not defined',
    }

## language/directive-prologue/func-decl-no-semi-parse.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/func-decl-no-semi-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "ReferenceError" is not defined',
    }

## language/directive-prologue/func-decl-not-first-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/func-decl-parse.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/func-decl-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "ReferenceError" is not defined',
    }

## language/directive-prologue/func-expr-final-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/func-expr-inside-func-decl-parse.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/func-expr-inside-func-decl-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "ReferenceError" is not defined',
    }

## language/directive-prologue/func-expr-no-semi-parse.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/func-expr-no-semi-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "ReferenceError" is not defined',
    }

## language/directive-prologue/func-expr-not-first-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ✅ Pass |  |

    'No error'

## language/directive-prologue/func-expr-parse.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ❌ Fail | An error occurred while loading the test from the `test262` folder. |
> | Transpile | ⬜ Skip | *This stage did not run.* |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    LoaderError {
      message: 'Negative tests, that is, test with the `negative` frontmatter attibute, are not supported at this time.',
    }

## language/directive-prologue/func-expr-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ✅ Pass |  |
> | Compile | ✅ Pass |  |
> | Execute | ❌ Fail | An error occurred while executing the test program. |

    Error {
      message: 'Uncaught ReferenceError: "ReferenceError" is not defined',
    }

## language/directive-prologue/get-accsr-inside-func-expr-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }

## language/directive-prologue/get-accsr-not-first-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }

## language/directive-prologue/get-accsr-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }

## language/directive-prologue/set-accsr-inside-func-expr-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }

## language/directive-prologue/set-accsr-not-first-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }

## language/directive-prologue/set-accsr-runtime.js

> | Test Stage | Status | Details |
> | :-- | :-- | :-- |
> | Load | ✅ Pass |  |
> | Transpile | ❌ Fail | An error occurred while transpiling the test with `@razuberi/transpiler`. |
> | Compile | ⬜ Skip | *This stage did not run.* |
> | Execute | ⬜ Skip | *This stage did not run.* |

    Error {
      message: 'No generator defined for ObjectExpression nodes.',
    }
